lib/grpc_kit/session/server_session.rb

```diff
@@ -176,7 +176,7 @@ def on_frame_recv(frame)
             stream.close_remote
           end
 
-          unless stream.inflight
+          if frame.end_stream? && !stream.inflight
```

Comment:
Check last data frame:
cf. https://nghttp2.org/documentation/types.html#c.nghttp2_on_frame_recv_callback

Reply:
I think this condition can be merged to above `if` clause as `elsif`.

Reply:
when using client_streamer (i think it is the same as using bidi_streamer), this server can't access data until a client sends END_STREAM flag. If the client sends 2 messages.  The server has to wait 1 min to handle these messages when the second message arrived 1 min after the first message.

Reply:
Currently, when gRPC client sends large data, grpc_kit server may receive multiple data frames, and an error occurs. 😢 

Do you have any good ideas  to collect chunk data?